README.md

```diff
@@ -6,3 +6,44 @@ This widget has two styles:
 - imitation of a real indicator.
 
 This widget allows you to set the style, the number of segments, pass the value for display, as well as get the current value (this method does not read the value from the images themselves). 
+
+# Build requirements
+
+ - CMake 3.5+ (CMake 3.12+ is recommended for more ergonomic `-DQNixieNumber_ROOT` support, see `examples/` for details);
+ - Qt5 / Qt6 (QNixieNumber will automatically links for the latest Qt version).
```

Comment:
`QNixieNumber automatically links to the latest` or `QNixieNumber will automatically link to the latest`

Also probably s/latest/latest found/